spec/index.bs

```diff
@@ -12706,13 +12723,18 @@ Issue: Because timestamp query provides high-resolution GPU timestamp, we need t
 
     Return `true` if the following requirements are met, and `false` if not.
 
-    - If |timestampWrites| is not [=list/is empty|empty=],
-        {{GPUFeatureName/"timestamp-query"}} must be [=enabled for=] |device|.
-    - No two entries in |timestampWrites| may have the same `location`.
-    - For each |timestampWrite| in |timestampWrites|:
-        - |timestampWrite|.`querySet` must be [$valid to use with$] |device|.
-        - |timestampWrite|.`querySet`.{{GPUQuerySet/type}} is {{GPUQueryType/"timestamp"}}.
-        - |timestampWrite|.`queryIndex` &lt; |timestampWrite|.`querySet`.{{GPUQuerySet/count}}.
+    - {{GPUFeatureName/"timestamp-query"}} must be [=enabled for=] |device|.
+    - |timestampWrites|.`querySet` must be [$valid to use with$] |device|.
+    - |timestampWrites|.`querySet`.{{GPUQuerySet/type}} must be {{GPUQueryType/"timestamp"}}.
+    - |timestampWrites|.`beginningOfPassWriteIndex` must be &lt; |timestampWrite|.`querySet`.{{GPUQuerySet/count}}.
+    - |timestampWrites|.`endOfPassWriteIndex` must be &lt; |timestampWrite|.`querySet`.{{GPUQuerySet/count}}.
```

Comment:
I'm guessing this is the text that the editorial comment means when it says some text may assume the value is undefined if it doesn't exist? Still feels a bit weird to says `undefined` is less than `count`.

Is it worth updating these statements to read something like "`beginningOfPassWriteIndex` must be `undefined` or < `count`"?

Reply:
er, no, it's other places in the spec that I didn't want to update right now. This is just a mistake, nice catch.